app/models/involved_civilian.rb

```diff
@@ -48,6 +50,10 @@ class InvolvedCivilian < InvolvedPerson
     validates :firearm_type, presence: true, subset: { in: FIREARM_TYPES }, if: :firearm?
   end
 
+  def k12_campus_incident?
+    incident.try(:on_k12_campus)
```

Comment:
Looks like i need to leave the try method in here, removing it results in many tests failing (false positives).